lstchain/data/lstchain_src_dep_config.json

```diff
@@ -1,84 +1,99 @@
 {
-  "events_filters": {
-    "intensity": [0, Infinity],
-    "width": [0, Infinity],
-    "length": [0, Infinity],
-    "wl": [0, 1],
-    "r": [0, 1],
-    "leakage_intensity_width_2": [0, 1]
-  },
+ "ReconstructionHDF5Writer": {
```

Comment:
I don't think we should change the existing config files here. 

We should create new config files per tool with only the configuration options for that tool.

This is also necessary, since `Tool` options need to be in the namespace of the Tool, so you anyway need config file per tool.

Reply:
Probably just keep the existing file like it is and add a new `lstchain/data/dl2_tool_config.json` with the correct config?

Using sensible defaults should make these config files rather small.